src/components/hosts/utils.ts

```diff
@@ -103,6 +103,16 @@ export const canDownloadHostLogs = (host: Host) =>
 export const canDownloadClusterLogs = (cluster: Cluster) =>
   !!(cluster.hosts || []).find((host) => canDownloadHostLogs(host));
 
+export const canAbortInstallation = (cluster: Cluster) => {
```

Comment:
This is cluster specific, it should go into `components/clusters/utils.ts

Reply:
oops